File: Labs/lab04/for.py
def sum_for(*args):
    result = 0
    for i in range(len(args)):
        result = result + args[i]

    return result

a = sum_for(1,2,3)
b = sum_for(0)
c = sum_for()
d = sum_for(1, -1, 1)
e = sum_for(10, -1, -1, -1)

print(a, b, c, d, e)

File: Labs/lab04/for.js
function sum_reduce(...args) {
    let result = 0;
    for(let i = 0; i < args.length; i++ ){
        result = result + args[i];
    }
    return result ;
}

const a = sum_reduce(1, 2, 3) ;
const b = sum_reduce(0) ;
const c = sum_reduce();
const d = sum_reduce(1, -1, 1) ;
const e = sum_reduce(10, -1, -1, -1) ;

console.log(a,b,c,d,e);